### PR TITLE
Added gitpod-init script

### PR DESCRIPTION
**Pull request by nickytonline**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

For some reason in the .gitpod.yml file, the `init` command does not work after the echo commands even with `&&`. The init command has been converted to a script instead.

## Related Tickets & Documents

#14697
#14696 (blocks this PR until it's merged)

## QA Instructions, Screenshots, Recordings

- Startup Gitpod for this branch, https://gitpod.io/#https://github.com/forem/forem/tree/nickytonline/fix-gitpod-init
- In the Forem Server terminal session, it completes successfully.

image

### UI accessibility concerns?

N/A build infrastructure

## Added/updated tests?

- [ ] Yes
- [x] No, and this is why: N/A build infrastructure
- [ ] I need help with writing tests

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

- [ ] I've updated the [Developer Docs](https://developers.forem.com) and/or
      [Admin Guide](https://admin.forem.com/), or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
- [ ] I've updated the README or added inline documentation
- [ ] I've added an entry to
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
- [ ] I will share this change in a [Changelog](https://forem.dev/t/changelog)
      or in a [forem.dev](http://forem.dev) post
- [ ] I will share this change internally with the appropriate teams
- [ ] I'm not sure how best to communicate this change and need help
- [x] This change does not need to be communicated, and this is why not: _please
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] What gif best describes this PR or how it makes you feel?

Jimmy Fallon motioning maybe?

### CODE REVIEW

File: .gitpod.yml [hunk 1]
```diff
@@ -16,12 +16,7 @@ tasks:
     before: |
       redis-server &
     init: >
-      cp .env_sample .env &&
-      echo "APP_DOMAIN=\"$(gp url 3000 | cut -f 3 -d /)\"" >> .env &&
-      echo 'APP_PROTOCOL="https://"' >> .env &&
-      gem install solargraph &&
```
Comment (nickytonline) [inline .gitpod.yml:22]: I have no idea why this fails to install solar graph and/or foreman, which cause bin/startup to fail or never run. The bash commands look fine and copy pasting them into a terminal work fine. Maybe it's a formatting issue in the YAML file? I honestly don't know. Regardless, a script seemed to make more sense.

File: gitpod-init.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+cp .env_sample .env;
```
Comment (djuber) [inline gitpod-init.sh:3]: **comment, non-blocking**: the semicolons are unneeded in the script - execution will proceed from top to bottom line by line automatically (I'm guessing you initially tried using separate statements in the init block in the yml file and this was a copy from an intermediate state).

**Review by djuber — APPROVED** [review]
double checked the script was set executable - makes sense

File: gitpod-init.sh (hunk 2 above)
Comment (nickytonline) [reply gitpod-init.sh:3]: I'll remove them. Thanks @djuber!

**Pull request merged**